src/ansys/mapdl/core/errors.py

```diff
@@ -49,52 +49,71 @@
 )
 
 
-class ANSYSDataTypeError(ValueError):
+## Abraham class
+class MapdlException(Exception):
+    def __init__(self, msg=""):
+        Exception.__init__(self, msg)
+
+
+## Main subclasses
+class MapdlValueError(MapdlException, ValueError):
+    def __init__(self, msg=""):
+        ValueError.__init__(self, msg)
+
+
+class MapdlFileNotFoundError(MapdlException, FileNotFoundError):
+    def __init__(self, msg=""):
+        FileNotFoundError.__init__(self, msg)
+
+
+class MapdlRuntimeError(MapdlException, RuntimeError):
+    """Raised when MAPDL passes an error"""
+
+    def __init__(self, msg=""):
+        RuntimeError.__init__(self, msg)
+
+
+## Inheritated
+class ANSYSDataTypeError(MapdlValueError):
     """Raised when and invalid data type is sent to APDLMath"""
 
     def __init__(self, msg=TYPE_MSG):
-        ValueError.__init__(self, msg)
+        MapdlValueError.__init__(self, msg)
```

Comment:
I'd probably convert all these to `super()` calls, regardless of whether or not this change is applied.

Reply:
I will do it in another PR :)

Reply:
See #2672 :)